command/agent/http.go

```diff
@@ -277,6 +277,11 @@ func handleUI(h http.Handler) http.Handler {
 
 func handleRootRedirect() http.Handler {
```

Comment:
Given that this handles the fallthrough of all other cases and returns 404, it might best to rename it.  Maybe `handleFallthrough` or `handleUnknown`?